### DISCUSSION TITLE
PR #4107

### DISCUSSION THREAD

**Discussion by beutlich** [modelica/ModelicaStandardLibrary#4139] [General]
This addressed the merged PR #4107 where Milestone is MSL 4.1.0, but PR was merged to main/4.0.x.

@TManikantan Please do:

- [x] Milestone needs to be fixed to MSL 4.0.1.
- [ ] Get familiar with the workflows mention in the [wiki](https://github.com/modelica/ModelicaStandardLibrary/wiki/Onboarding-MAP-Lib-Project-Leader-and-Deputy#msl-github-issues-and-pull-requests)
- [x] Usually PRs target the master branch and are back-ported to maintenance banches if necessary. This PR is now missing in master. Should it be cherry-picked to master?

**Comment by TManikantan** [comment]
@beutlich we missed it, thankyou for noticing and pointing it out. We have created a branch from the master and cherry picked the commits  and created a PR(#4140) to merge it back to master. I believe this would solve the problem.
